test/test_runner.go

```diff
@@ -695,7 +651,20 @@ func (r *TestRunner) parseAndCheckImport(
 
 	code, err := r.importResolver(location)
 	if err != nil {
-		return nil, nil, err
+		addressLocation, ok := location.(common.AddressLocation)
+		if ok {
+			// System-defined contracts are obtained from
+			// the blockchain.
+			account, err := r.blockchain.GetAccount(
+				flow.Address(addressLocation.Address),
+			)
+			if err != nil {
+				panic(err)
+			}
```

Comment:
instead of panicking, maybe return this error

Reply:
Good catch :+1: 
Fixed in https://github.com/onflow/cadence-tools/pull/205/commits/3b60ad4a5f96051465b9954f9aa0f5ebbf2eb794